CONTRIBUTING.md

```diff
@@ -89,6 +90,67 @@ You generally should not merge your own registrations (though you can make reque
 * [write] You can merge improvements to the README, these guidelines, or our workflows.
 * [admin] You can give other contributors triage-level access so they can apply labels to PRs, or write-level permissions to merge PRs.
 
+## Appendix: Checking if a repository contains all registered versions of a package
+
+When someone wishes to move a package from one repo to another, it is important that the new repo contains all of the tree hashes corresponding to registered versions of a package. That way these old versions of the package can continue to be installed from the new repository. In order to check if a given repository contains all of the registered versions of a package, the following script can be used:
+
+```julia
+using RegistryInstances, UUIDs, Git
+
+const GENERAL_UUID = UUID("23338594-aafe-5451-b93e-139f81909106")
+
+pretty_print_row(row) = println(row.pkg_name, ": v", row.version, " ", row.found ? "found" : "is missing")
+pretty_print_table(table) = foreach(pretty_print_row, table)
+
+function check_all_found(table)
+    idx = findfirst(row -> !row.found, table)
+    idx === nothing && return nothing
+    row = table[idx]
+    error(string("Repository missing v", row.version, " of package $(row.pkg_name)"))
+end
+
+function check_packages_versions(pkg_names, repo_url; registry_uuid=GENERAL_UUID, verbose=true, throw=true)
+    dir = mktempdir()
+    run(`$(git()) clone $(repo_url) $dir`)
+
+    registry = only(filter!(r -> r.uuid == registry_uuid, reachable_registries()))
+
+    table = @NamedTuple{pkg_name::String, version::VersionNumber, found::Bool}[]
+
+    for pkg_name in pkg_names
+        pkg = registry.pkgs[only(uuids_from_name(registry, pkg_name))]
+        versions = registry_info(pkg).version_info
+        for version in sort(collect(keys(versions)))
+            tree_sha = versions[version].git_tree_sha1
+            found = success(`$(git()) -C $dir rev-parse -q --verify "$(tree_sha)^{tree}"`)
+
+            push!(table, (; pkg_name, version, found))
+        end
+    end
+    verbose && pretty_print(table)
```

Comment:
Was this meant to be
```suggestion
    verbose && pretty_print_table(table)
```
?

Reply:
Oops, yes

Reply:
I had tested it, just not in a fresh session, and I still had an old method in my session. Tested now in a fresh session & it should be OK